windows/src/desktop/kmshell/main/initprog.pas

```diff
@@ -337,6 +337,34 @@ procedure Run;
   end;
 end;
 
+function DoCheckTIPInstallStatus(FSilent: Boolean): Boolean;
+begin
+(*
+  Note: this needs a significant refactor because enumerating input processor
+        profiles under Administrator account may return different results to an
+        end user account. It is also much less likely that a keyboard will not
+        be available with Keyman 14's keyboard registration model. So we may be
+        able to avoid using this altogether.
+
+        For now, we will disable this check here and in the future may re-enable
+        it and rewrite it if it proves necessary to improve profile stability.
+
+        See #4893.
+  if not TKeyboardTIPCheck.CheckKeyboardTIPInstallStatus then
+  begin
+    if FSilent then
+      Exit(False);
+
+    if MessageDlg('Some keyboard profiles have been damaged. Correct these now?', mtConfirmation, mbOkCancel, 0) = mrOk then
+    begin
+      WaitForElevatedConfiguration(0, '-repair', True);
+      TKeyboardTIPCheck.ReEnablePostRepair;
```

Comment:
Where'd this line - the `ReEnablePostRepair` - coming from?  The rest is pretty clearly refactored from old code visible in this PR's changeset, but this line lacks any corresponding 'old code' source.

Reply:
Yeah, I should have deleted that line again after attempting a fix with that function (since deleted) -- it's commented out so I won't worry about it for now. A reminder that I'll need to do something to that effect if we need this code in the future.

Reply:
Mind adding an extra single-line comment mark on this line in particular, then?